appveyor.yml

```diff
@@ -20,8 +25,9 @@ shallow_clone: true
 clone_depth: 1
 test_script:
   - node --version
-  - npm --version
+  - yarn --version || npm --version
```

Comment:
I like that!

The alternative, if I'm not mistaken (assuming AppVeyor is running a Windows commandline and not a Posixy one), would be `yarn --version & npm --version` (`&` in `cmd` is identical to `;` in `sh`); however, somebody might see that and try to split it into two commands without realizing the first one needs its result suppressed; this is clearer on that point.